ArrayOps.java

```diff
@@ -1,26 +1,201 @@
+        for (int i= 0; i < array.length; i++)
+        {
+            if( array[i] > max )
+            {
+                max =array[i]; 
+            }
+        }
+        //secondMax = array[0];
+        for (int i= 0; i < array.length; i++)
+        {   
+            if (array[i]== max)
+            {
+                counter++; 
+            }
+            else
+            {
+               if ( array[i]> secondMax )
+               {
+                secondMax = array[i]; 
+               }
+            }
+            
+        }
+        if ( counter > 1)
+        {
+            return max;
+        }
+        else
+        {
+            return secondMax;
+        }
+        
     }
+    
 
-    public static boolean containsTheSameElements(int [] array1,int [] array2) {
-        // Write your code here:
-        return false;
+    public static int[] comparelong(int [] array1, int[] array2 )
+    {
+        int length1 = array1.length; 
+        int length2 = array2.length; 
+        if ( length1 > length2)
+        {
+             return array1; 
+        }
+        else
+        {
+            return array2;
+        }
     }
+    public static int[] compareShort(int[] array1, int[] array2 )
+    {
+        int length1 = array1.length; 
+        int length2 = array2.length; 
+        if ( length1 < length2)
+        {
+             return array1; 
+        }
+        else
+        {
+            return array2;
+        }
+    }
+
+
+
+    public static boolean containsTheSameElements(int [] array1,int [] array2) 
+    {
+        boolean find = false ; 
+        int [] longArr = comparelong(array1,array2); 
+        int [] shortArr = compareShort(array1,array2);
+        int counter = 0; 
+        int num =0 ; 
+        
+            while (counter < longArr.length)
+            {
+            num= longArr[counter]; 
+             for (int i =0; i < shortArr.length ; i ++)
+             {
+                if (num == shortArr[i])
+                {
+                   find= true; 
+                }
+             }
+             if ( find != true)
+             {
+                return false; 
+                
+             }
+             else {
+                find = false; 
+             }
+             counter ++;
+            }
+
+            return true; 
+
+        }  
+    
+    
+
+    public static boolean isSorted(int [] array) 
+    {
+        boolean up = true;
+        boolean down = true;
+        for ( int i =0 ; i < array.length -1; i ++ )
+        {
+            if (up == true)
+            {
+            if ( i <= (array.length - 1))
+             {
+                if ( array[i] <= array[i+1])
+                 {
+                 up = true;
+                 }
+                 else                      
+                 {
+                  up = false;
+                 }
+             } 
+             else {
+                up = true;
+             } 
+            } 
+        }
+        
```

Comment:
very complex and unreadable code, you should have done 2 helper functions that will check if increasing or decreasing instead of this very long function.
also why if ( i <= (array.length - 1))? it is true because of the for.
if -else inside if inside if, not fun to have.